pkg/agents/team5/agent.go

```diff
@@ -132,12 +161,14 @@ func (a *CustomAgent5) Run() {
 				log.Error("Simulation - team5/agent.go: \t Impossible error reached")
 			}
 		}
+
 		a.lastMeal = lastMeal
 		if a.lastMeal > 0 {
 			a.daysSinceLastMeal = 0
 		}
-		a.updateSatisfaction()
 		a.hpAfterEating = a.HP()
-		//a.messagingCounter = 0
+		a.updateSatisfaction()
```

Comment:
Does this function call need to be removed?

Reply:
Yeah, satisfaction should be updated only once at the beginning of the block where the agent tries to eat so this one should be removed

Reply:
Thank you for removing